code/addons/onboarding/src/components/Confetti/Confetti.tsx

```diff
@@ -1,4 +1,4 @@
-import ReactConfetti from 'react-confetti';
+import { ReactConfetti } from 'react-confetti/src/ReactConfetti';
```

Comment:
Can you elaborate why we have to deep-link?

Reply:
Because it doesn't expose ESM, but they do expose their source...
So taking the source causes better minification, and no `can't find react` issues.